installer/pkg/config-generator/tls.go

```diff
@@ -123,8 +136,95 @@ func (c *ConfigGenerator) GenerateTLSConfig(clusterDir string) error {
 		Validity:     validityThreeYears,
 		IsCA:         false}
 
-	if _, _, err := generateCert(clusterDir, kubeCAKey, kubeCACert, ingressKeyPath, ingressCertPath, cfg); err != nil {
-		return fmt.Errorf("failed to generate ingress CAs: %v", err)
+	if _, _, err = generateCert(clusterDir, kubeCAKey, kubeCACert, ingressKeyPath, ingressCertPath, cfg); err != nil {
```

Comment:
why remove the `:` from here? we can happily keep it like we do in the rest of the instances above